Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.centos.org/centos/centos:7
+FROM registry.access.redhat.com/ubi8/python-36:latest
```

Comment:
just changing the base image wont work out @mbharatk you might have to manually build and verify or by trial and error method if it works PTAL